lib/roo_on_rails/checks/heroku/app_exists.rb

```diff
@@ -16,6 +16,10 @@ module Heroku
       # Output context:
       # - heroku.app.{env}: an app name.
       class AppExists < EnvSpecific
+        ACCEPTABLE_ENV_NAMES = {
+          'production' => ['prod', 'production'],
+          'staging'    => ['stg', 'staging']
```

Comment:
@halfwolf
- stg, staging, prd, and production work for me
- perhaps we can leave the 'uniformity' check - it's a nice-to-have but perhaps not super-important